src/biogeophys/SoilTemperatureMod.F90

```diff
@@ -734,11 +734,16 @@ subroutine SoilThermProp (bounds, num_urbanc, filter_urbanc, num_nolakec, filter
                endif
             endif
 
-            ! Thermal conductivity of snow, which from Jordan (1991) pp. 18
+            ! Thermal conductivity of snow, which from Sturm (1997)
             ! Only examine levels from snl(c)+1 -> 0 where snl(c) < 1
             if (snl(c)+1 < 1 .AND. (j >= snl(c)+1) .AND. (j <= 0)) then  
-               bw(c,j) = (h2osoi_ice(c,j)+h2osoi_liq(c,j))/(frac_sno(c)*dz(c,j))
-               thk(c,j) = tkair + (7.75e-5_r8 *bw(c,j) + 1.105e-6_r8*bw(c,j)*bw(c,j))*(tkice-tkair)
+               bw(c,j) = ((h2osoi_ice(c,j)*1)+h2osoi_liq(c,j))/(frac_sno(c)*dz(c,j)) ! ==RHOS
+               ! thk(c,j) = tkair + (7.75e-5_r8 *bw(c,j) + 1.105e-6_r8*bw(c,j)*bw(c,j))*(tkice-tkair) ! Original (Jordan) Parameterisation
+               if (bw(c,j)  <= 156) then !LMW or 0.156 ?
```

Comment:
Here is where I think we'll want to use a namelist option to chose between Jornand and Sturm formulations for `thk`